sklearn/tests/test_dummy.py

```diff
@@ -220,6 +220,19 @@ def test_mean_strategy_regressor():
     assert_array_equal(reg.predict(X), [np.mean(y)] * len(X))
 
 
+def test_dummy_regressor_zero_std():
+
```

Comment:
rm blank line, please

Reply:
They are all over this file, so I'll go ahead and remove them everywhere.